web-local/src/common/data-modeler-service/TableActions.ts

```diff
@@ -376,13 +375,17 @@ export class TableActions extends DataModelerActions {
     args: PersistentTableStateActionArg,
     tableName: string
   ) {
-    await runtimeServiceMigrateDelete(
-      this.databaseService.getDatabaseClient().getInstanceId(),
-      {
-        name: tableName,
-      }
-    );
-    return this.dropTable(args, tableName);
+    // MigrateDelete has been deprecated. All migrations should go through reconciliation.
```

Comment:
Good observation. Will make sure to fix this in the artifacts UI changes branch.

Reply:
This code path here was only used in the Node CLI, so shouldn't matter since we're replacing that